src/components/Form/Form.tsx

```diff
@@ -4,7 +4,7 @@ export interface FormProps extends FormHTMLAttributes<HTMLFormElement> {
   children: ReactNode;
 }
 
-const Form = ({ children, onInvalid, ...formAttrs }: FormProps): JSX.Element => {
+const Form = ({ children, onInvalid, className, ...formAttrs }: FormProps): JSX.Element => {
```

Comment:
not showing as a control in SB

Reply:
looks like this applies spacing between the form title/description (h2 & p) and the form element. it felt weird to me at first that it didn't apply above the title, but after thinking about it i think it's right. let's confirm with catie that this is her expected behavior too.

Reply:
confirmed with catie this is expected behavior and we are good to resolve this comment